### PR TITLE
New version: OliverBetz.ExifTool version 12.73

### PR DESCRIPTION
**Pull request by fancybody**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135510)

### CODE REVIEW

**Comment by fancybody** [comment]
@stephengillie
The pipeline seems to have stopped abrubtly.
Can you please restart it?

**Review by stephengillie — DISMISSED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -2147012867 2024-01-26 21:46:44.939 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(267)\WindowsPackageManager.dll!00007FFD5B3FA384: (caller: 00007FFD5B3FAD01) Exception(1) tid(1e8) 80D02002  2024-01-26 21:47:12.520 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!00007FFD5B3F395E: (caller: 00007FFD5B3F5C49) Exception(2) tid(1e8) 80072EFD     Msg:[InternetOpenUrl() failed.]  2024-01-26 21:47:12.633 [CLI ] Failed to download, waiting a bit and retry. Url: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_12.73_64.exe 2024-01-26 21:48:13.177 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(267)\WindowsPackageManager.dll!00007FFD5B3FA384: (caller: 00007FFD5B3FAD01) Exception(3) tid(1e8) 80D02002  2024-01-26 21:48:40.192 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!00007FFD5B3F395E: (caller: 00007FFD5B3F5C49) Exception(4) tid(1e8) 80072EFD     Msg:[InternetOpenUrl() failed.]  2024-01-26 21:48:40.318 [CLI ] Caught wil::ResultException: D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!00007FFD5B3F395E: (caller: 00007FFD5B3F5C49) Exception(4) tid(1e8) 80072EFD     Msg:[InternetOpenUrl() failed.]  2024-01-26 14:52:42.301   Exception message:                           Internal error: Cannot expand "autopf64" constant on this version of Windows. Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\ExifTool.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\ExifTool.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\exiftool_files\perl.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\exiftool_files\perl.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform. Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\Desktop\exiftool(-k).exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.)  ---> System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'C:\Users\Validator\Desktop\exiftool(-k).exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.

(Automated response - build 670.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -2147012867 2024-01-26 21:46:44.939 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(267)\WindowsPackageManager.dll!00007FFD5B3FA384: (caller: 00007FFD5B3FAD01) Exception(1) tid(1e8) 80D02002  2024-01-26 21:47:12.520 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!00007FFD5B3F395E: (caller: 00007FFD5B3F5C49) Exception(2) tid(1e8) 80072EFD     Msg:[InternetOpenUrl() failed.]  2024-01-26 21:47:12.633 [CLI ] Failed to download, waiting a bit and retry. Url: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_12.73_64.exe 2024-01-26 21:48:13.177 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(267)\WindowsPackageManager.dll!00007FFD5B3FA384: (caller: 00007FFD5B3FAD01) Exception(3) tid(1e8) 80D02002  2024-01-26 21:48:40.192 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!00007FFD5B3F395E: (caller: 00007FFD5B3F5C49) Exception(4) tid(1e8) 80072EFD     Msg:[InternetOpenUrl() failed.]  2024-01-26 21:48:40.318 [CLI ] Caught wil::ResultException: D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Downloader.cpp(122)\WindowsPackageManager.dll!00007FFD5B3F395E: (caller: 00007FFD5B3F5C49) Exception(4) tid(1e8) 80072EFD     Msg:[InternetOpenUrl() failed.]  2024-01-26 14:52:42.301   Exception message:                           Internal error: Cannot expand "autopf64" constant on this version of Windows. Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\ExifTool.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\ExifTool.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\exiftool_files\perl.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\ExifTool\exiftool_files\perl.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform. Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\Desktop\exiftool(-k).exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.)  ---> System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'C:\Users\Validator\Desktop\exiftool(-k).exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.

(Automated response - build 670.)

File: manifests/o/OliverBetz/ExifTool/12.73/OliverBetz.ExifTool.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  Scope: machine
+  ElevationRequirement: elevatesSelf
+  InstallerUrl: https://oliverbetz.de/cms/files/Artikel/ExifTool-for-Windows/ExifTool_install_12.73_64.exe
+  InstallerSha256: B1862210C9BABBC6F3A85B3F3B7D4A190F630788F6D30C5559FFB105EAD9CDD8
+  InstallerSwitches:
+    Custom: /ALLUSERS
+  AppsAndFeaturesEntries:
+  - DisplayName: ExifTool version 12.73_32
+    DisplayVersion: 12.73_32
+    ProductCode: ExifTool_is1
```
Comment (stephengillie) [inline manifests/o/OliverBetz/ExifTool/12.73/OliverBetz.ExifTool.installer.yaml:30]:
```suggestion
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**